stl/inc/memory_resource

```diff
@@ -356,8 +356,7 @@ namespace pmr {
             void* const _Ptr                 = _Resource->allocate(_Bytes, _Align);
             _Check_alignment(_Ptr, _Align);
 
-            _Oversized_header* const _Hdr =
-                reinterpret_cast<_Oversized_header*>(reinterpret_cast<char*>(_Ptr) + _Bytes) - 1;
+            _Oversized_header* const _Hdr = reinterpret_cast<_Oversized_header*>(static_cast<char*>(_Ptr) + _Bytes) - 1;

```

Comment:
https://github.com/microsoft/STL/pull/2096 has the same change here and below...

Reply:
Exactly the same, so if both changes accepted, would merge smooth